include/openssl/core_names.h

```diff
@@ -191,6 +191,12 @@ extern "C" {
 #define OSSL_PKEY_PARAM_FFC_P        "p"
 #define OSSL_PKEY_PARAM_FFC_G        "g"
 #define OSSL_PKEY_PARAM_FFC_Q        "q"
+#define OSSL_PKEY_PARAM_FFC_GINDEX   "gindex"
+#define OSSL_PKEY_PARAM_FFC_PCOUNTER "pcounter"
+#define OSSL_PKEY_PARAM_FFC_SEED     "seed"
+#define OSSL_PKEY_PARAM_FFC_COFACTOR "j"
+#define OSSL_PKEY_PARAM_FFC_H        "hindex"
+#define OSSL_PKEY_PARAM_FFC_GROUP    "group"
```

Comment:
Some of these seem to remain undocumented...  I'm not sure I understand if that's as it should be or not, so see this as a ping in case it's an omission, otherwise feel free to ignore this.

Reply:
It will be done - just not here..
The entire core_names.h will need to be reviewed at some point, there is undocumented stuff all over the place.

Reply:
Okie